jar-infer/jar-infer-cli/build.gradle

```diff
@@ -12,11 +12,11 @@ repositories {
 }
 
 dependencies {
-    compile deps.build.commonscli
-    compile deps.build.guava
+    implementation deps.build.commonscli
```

Comment:
After this change, I checked the shadow jar file installed after running `./gradlew publishToMavenLocal`, and the set of files is identical to before the change.  Also the generated POM file seems to be identical.